scripts/new_project.kts

```diff
@@ -38,14 +39,32 @@ object NewProject {
         println("\n${message}\n")
     }
 
-    private fun copyFiles(fromPath: String, toPath: String) {
-        val targetFolder = File("../$toPath")
-        val sourceFolder = File("../$fromPath")
+    private fun copyFiles(fromPath: String) {
+        val targetFolder = File(getRootPathOfNewProjectFolder())
+        val sourceFolder = File(getRootPathOfNewProjectFolder().replace(appNameWithoutSpace, "") + fromPath)
```

Comment:
I think we should keep the old implementation instead since we may re-use this function for other purposes. Currently it's hard for setting to copy files from a specific folder to another folder 😅  For example, my case is to copy files from the old package path `co.nimblehq.coroutine` to a new package path `com.test.abc`.